src/components/Cards/PlaylistVideoCard.jsx

```diff
@@ -0,0 +1,25 @@
+import "./card.css"
+import {useVideos} from "../../context/video-context";
```

Comment:
Suggestion: use absolute path instead of relative path. Makes code look clean and readable.

Reply:
Okay will do it.